DigitalLearningSolutions.Data.Tests/DataServices/CentresDataServiceTests.cs

```diff
@@ -184,6 +184,49 @@ public void UpdateCentreManagerDetails_updates_centre()
             }
         }
 
+        [Test]

```

Comment:
I think we plan to remove the Data.Tests project from the solution. Could we test the Controller and/or and public service methods added to the Web project in Web.Tests instead?